util/data_generator.py

```diff
@@ -107,7 +107,6 @@ def get_sha3_fixed(self):
         ct = [x for x in ct_bytes]
 
         del (sha3_fixed)
-        self.advance_fixed()
         return pt, ct, key
```

Comment:
Post-merge, sorry.

It looks like `key` output is never used in `capture_sha3.py`, so we could probably get rid of it here.